libvast/test/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set_tests_properties(build-vast-test PROPERTIES FIXTURES_SETUP
 # Helper macro to construct a CMake test from a VAST test suite.
 macro (make_test suite)
   string(REPLACE " " "_" test_name ${suite})
-  add_test(NAME ${test_name} COMMAND vast-test -v 3 -r 600 -s "^${suite}$"
+  add_test(NAME ${test_name} COMMAND vast-test -v 4 -r 600 -s "^${suite}$"
```

Comment:
The unit-tests usually finish below 10 seconds in debug mode on my laptop, shouldn't we lower the timeout a bit?

Reply:
I am open to changing it to maybe a minute to account for debug builds and random CI hangs.

Reply:
I pushed that change, please check back.